lib/Lib.IntTypes.fst

```diff
@@ -374,6 +742,13 @@ let eq_mask #t a b =
   | U32  -> UInt32.eq_mask a b
   | U64  -> UInt64.eq_mask a b
   | U128 -> UInt128.eq_mask a b
+  | S8   -> to_i8 #U8 #SEC (UInt8.eq_mask (to_u8 a) (to_u8 b))
```

Comment:
Can we define a more efficient `eq_mask` operator for `signed` integers using `shift_arithmetic_right`?